qdao/circuit.py

```diff
@@ -109,7 +112,99 @@ def run(self, circuit: Any) -> List[QdaoCircuit]:
         if instrs:
             sub_circ = self._circ_helper.gen_sub_circ(instrs, self._nl, self._np)
             sub_circs.append(sub_circ)
+        print("----------StaticPartitioner-----------")
+        print("num of sub-circuits:" + str(len(sub_circs)))
+        return sub_circs
+
+
+class dptask:
+    """To assist the Uniq partitioning algorithm"""
+
+    def __init__(self, qubitNum: int, gateNum: int):
+        self.gateNum = gateNum
+        self.qubitNum = qubitNum
+
+    def addGate(self, gateIndex: int, gatePos):
```

Comment:
Similarly, please refer to the [style guide](https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations). 

In short
- For classes, use [CamelCase](https://peps.python.org/pep-0008/#naming-conventions)  convention.
- For methods, variables, use `lower_case_with_underscore` convention.